debian/kano-updater.install

```diff
@@ -14,3 +14,4 @@ locale/* /usr/share/locale
 
 conf/apt etc/
 conf/dhcpcd-hooks lib/dhcpcd/
+conf/systemd usr/lib/
```

Comment:
should this be `conf/ usr/lib`, in order to land at `/usr/lib/systemd/system`?

Reply:
The above actually says that the `conf/systemd` folder (and everything in it) will be installed at `/usr/lib/`. Each folder in `conf/` has a slightly different install path.

Reply:
You're right, sorry